.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - repo: https://github.com/dnephin/pre-commit-golang
+    rev: v0.5.0
```

Comment:
Are there any specific considerations necessary for this remote source being used to operate on workstations? A Git repository that is using a mutable Git tag as a source (without other verification) can become compromised and used in supply chain type attack vectors, for example.

I'm not sure if the HashiCorp security team has any specific guidance with the `pre-commit` development tooling, but some remediations to prevent this type of concern may be any of these potential options:

* Forking the remote source into the `hashicorp` GitHub organization
* Using a Git commit SHA as the revision instead of a tag
* Using a `local` repo and calling locally-owned Makefile targets, shell scripts, or `go`+`git diff` commands directly as appropriate

Thanks!